_posts/2016-03-07-2016-forman-survey-analysis.md

```diff
@@ -0,0 +1,342 @@
+respect to a particular platform. We do well because of our neutrality.
+
+![Taxonomies](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/taxonomies.png)
+
+One of our more advanced features, Locations / Organisations are a scoping
+mechanism, useful if you have resources that should only matter to certains
+places or people (e.g. multi-tenant setups). Whats interesting here is that
+while the number of users not using Locs/Orgs is constant, many of those who
+used just one last year now use both (24% -> 33%).
+
+We actually asked three questions on this feature, as the developer community
+wanted to gather usecases for a potential refactoring of the feature, and
+wanted to be sure we didn't break it for anyone. We'll be able to say more on
+this once the refactoring is decided on (or not)
+
+This next graph is huge. Sorry ;)
+
+![Plugin Usage](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/plugin-usage.png)
+
+Most of the big usage numbers here are entirely expected. Discovery tops the
+chart again, but it's good to see Katello move in to second place; closely
+following are foreman\_hooks, puppetdb\_foreman, foreman\_templates and
+foreman\_bootdisk. I'll admit I was surprised to see foreman\_dhcp\_browser up at
+30% - perhaps this indicates a need for more Smart Proxy visibility from within
+the UI? Happily, such things are in progress!
+
+The fast movers are foreman\_ansible, and foreman\_openscap, both developed this
+year - in particular the Ansible plugin is only a few months old, so it's 13%
+uptake is rapid indeed.
+
+As with last year, this shows some of the activity in our plugin ecosystem,
+which is great to see. We really want to see plugins succeed, and provide a lot
+of materials to help get started writing them, so graph like these are a reward
+for that effort.
+
+## <a name="upcoming-work"></a>Upcoming work
+
+For this, we changed the style this year. Rather than ask for interest ratings
+on individual features from 1-5 (which can be a problem since not every feature
+is of interest to all users), we decided to simply ask for your top 3 wishes,
+in no specific order. This list was drawn from the items still not done from
+last year, from some things that come up over and over in conversations, and so
+on.
+
+![Future Features](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/future-features.png)
+
+Overwhelmingly, Puppet 4 support tops the bill. We knew this would be high on
+the list, so we took time to meet with Puppetlabs at CfgMgmt Camp, and come up
+with a plan of action (you can read it [here][puppet4]). This work is
+*tentatively* scheduled for 1.12, but as ever with open source, there's no
+promises.
+
+Following that, we see better container support coming in strong. We've already
+got a Docker plugin, Docker registry support via Pulp/Katello, and CoreOS &
+Atomic provisioning support. There's more we can do though, so do file
```

Comment:
I don't want to incite a debate directly here, but I've reworded the paragraph to say "hey we already have this stuff, was there anything else?"